toolchain/docs/lower.md

```diff
@@ -23,3 +24,26 @@ can first generate type information for function arguments.
 Lowering is done per `SemIR::InstBlock`. This minimizes changes to the
 `IRBuilder` insertion point, something that is both expensive and potentially
 fragile.
+
+## Mangling
+
+Part of lowering is choosing deterministically unique identifiers for each
+lowered entity to use in platform object files. Any feature of an entity (eg:
+the namespace it appears in, parameters for overloaded functions, etc) that
+would create a distinct entity must be included in some way in the generated
+identifier.
+
+The current rudimentary name mangling scheme is as follows:
+
+-   Start with `_C`
+
+-   Then the unqualified function name (function name mangling is the only thing
+    implemented at the moment)
+
+-   `.` separated scopes (namespaces/classes), most nested first, outermost last
+
+-   or, if the function is in an `impl`:
+
+    -   the implementing type, per the scope mangling above
+
+    -   the interface type, per the scope mangling above
```

Comment:
The vertical whitespace makes each of these a separate one-item list.

```suggestion
-   Start with `_C`.
-   Then the unqualified function name (function name mangling is the only thing
    implemented at the moment).
-   Then `.` separated scopes (namespaces/classes), most nested first, outermost last.
-   Or, if the function is in an `impl`:
    -   the implementing type, per the scope mangling above.
    -   the interface type, per the scope mangling above.
```

(I also suggested some minor copyedits while I was there)

Reply:
> The vertical whitespace makes each of these a separate one-item list.

Oh - I had looked for an example of sub lists, and used that as the basis for this - it was here: https://docs.carbon-lang.dev/toolchain/docs/parse.html#optional-modifiers-before-an-introducer where there are blank lines between the "steps" in this list. But perhaps that's intentionally different there?